Dockerfile

```diff
@@ -135,3 +190,11 @@ COPY --from=image-ca-certificates / /
 LABEL org.opencontainers.image.source https://github.com/siderolabs/discovery-service
 ENTRYPOINT ["/discovery-service"]
 
+FROM scratch AS image-snapshot-decoder
```

Comment:
we need to disable this image in Kres I guess

Reply:
bump this!

Reply:
done

Reply:
also needs a rekres now, since we changed the builders

Reply:
done